src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    private double width;
+    private double height;
+
+    public Rectangle(String color, double width, double height) {
+        super(color);
+        this.height = height;
+        this.width = width;
+    }
+
+    @Override
+    public double getArea() {
+        return height * width;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: rectangle, area: "
```

Comment:
Avoid using System.out.println in your draw method. Instead, consider returning the String representation of the object and let the caller decide how to handle it.